andromeda/README.md

```diff
@@ -101,3 +101,39 @@ Calculate weights for the given image coordinates.
     }
     ```
 
+### Fetch iNaturalist observations as JSON data
+Read all observations for iNaturalist user and return as JSON.
+- GET __/api/inaturalist/<inat_user>__
+  - Output
+    ```  
+    {
+      "data": [
+        {
+          "Date": "...",
+          "Image_Label": "...",
+          "Image_Link": "...",
+          "Lat": 42.9349293461,
+          "Long": -88.0380119262,
+          "Place": "...",
+          "Seconds": 57785,
+          "Species": "White-tailed Deer",
+          "Time": "...",
+          "User": "..."
+        },
+        ...
+      "user_id": "...",
+      "warnings": [
+        "missing_lat_long"
+      ]
+    }
+    ```
+
+### Fetch iNaturalist observations as CSV data
+Read all observations for iNaturalist user and return as a CSV file.
+- GET __/api/inaturalist/<inat_user>?format=csv__
+  - Output
+    ```  
+    Image_Label,Image_Link,Species,User,Date,Time,Seconds,Place,Lat,Long
```

Comment:
This looks like there will be no warnings, or at least no record-level warning(s), in contrast to the JSON output. I don't think that's necessarily bad or needs to be fixed, but because there is a contrast, it should probably be documented?

Reply:
I added a note:
https://github.com/Imageomics/Andromeda/blob/ceef24a041c9eec09b383ab023d3a415746a05ad/andromeda/README.md?plain=1#L140